docs/docs/guides/testing.md

```diff
@@ -76,7 +76,7 @@ A common pattern for atoms is using asynchronous queries fetch the state of the
 function flushPromisesAndTimers(): Promise<void> {
   return act(
     () =>
-      new Promise(resolve => {
+      new Promise((resolve) => {
```

Comment:
For consistency, our prettier configuration does not wrap single parameters with parens.